src/resource/resource-opening-hours/opening-period/OpeningPeriod.tsx

```diff
@@ -28,12 +28,14 @@ export default function OpeningPeriod({
   datePeriodConfig,
   language,
   deletePeriod,
+  initiallyOpen = false,
```

Comment:
I think initiallyOpen can be mandatory prop 🤔

Reply:
- [ ] Change initiallyOpen to a mandatory prop